interface/src/app/home/home.js

```diff
@@ -2,6 +2,7 @@
  * adage.home module.
  */
 angular.module('adage.home', [
+  'ngSanitize',
```

Comment:
It looks as if `ngSanitize` is not used anywhere. Am I missing something?

Reply:
According to:
https://docs.angularjs.org/api/ngSanitize/service/$sanitize
`ng-bind-html` seems to automatically call `$sanitize`, which requires `ngSanitize`.

Reply:
I tested the example at the end of this page:
https://docs.angularjs.org/api/ng/directive/ngBindHtml
If `ngSanitize` module is removed from dependency, the page won't be rendered correctly. Here is the error:
```
Error: [$sce:unsafe] http://errors.angularjs.org/1.6.9-build.5530+sha.07d84dd/$sce/unsafe
    at https://code.angularjs.org/snapshot/angular.min.js:7:93
    at f (https://code.angularjs.org/snapshot/angular.min.js:157:225)
    at getTrusted (https://code.angularjs.org/snapshot/angular.min.js:158:462)
    at Object.c.(anonymous function) [as getTrustedHtml] (https://code.angularjs.org/snapshot/angular.min.js:160:175)
    at https://code.angularjs.org/snapshot/angular.min.js:286:459
    at m.$digest (https://code.angularjs.org/snapshot/angular.min.js:148:416)
    at m.$apply (https://code.angularjs.org/snapshot/angular.min.js:151:505)
    at https://code.angularjs.org/snapshot/angular.min.js:22:117
    at Object.invoke (https://code.angularjs.org/snapshot/angular.min.js:44:442)
    at c (https://code.angularjs.org/snapshot/angular.min.js:22:38)
```

Reply:
👍 

Since that connection is not obvious, can we put a short comment at line 5 saying something like:

```javascript
  'ngSanitize',  // used by ng-bind-html
```